python/rmm/_lib/memory_resource.pyx

```diff
@@ -117,29 +117,52 @@ cdef class FixedSizeMemoryResource(MemoryResource):
 cdef class BinningMemoryResource(MemoryResource):
     def __cinit__(
         self,
-        MemoryResource upstream_mr
+        MemoryResource upstream_mr,
+        min_size_exponent=None,
+        max_size_exponent=None,
```

Comment:
These objects should be typed and if possible it would be good to use `-1` or something similar instead of `None` so we can avoid unnecessary Python objects.

Reply:
Please review on the binning_memory_resource PR where these changes live. #457